### ISSUE TITLE
Error in IE7 when hitting F5

### ISSUE THREAD

**Issue by olemarius** [Schepp/box-sizing-polyfill#9]
When a page containing an element using boxsizing in IE7 loads, there's no errors showing, but once hitting F5 the following error shows up in the console (IE9 in IE7 mode)

`SCRIPT65535: Unexpected call to method or property access. 
boxsizing.htc, line 99 character 4`

Not sure what this is, but I'll report it even thought the script is still initialized and working (except from another bug I'll file separately)

**Comment by olemarius** [comment]
```
function restore() {
    if (apply) {
        element.runtimeStyle.removeAttribute("width"); // THIS IS LINE 99
        element.runtimeStyle.removeAttribute("height");
    }
}
```

**Comment by TheKidCoder** [comment]
Any progress/trace as to why this error is being thrown?

Really trashing up my console.

**Comment by olemarius** [comment]
TheKidCoder: I think my errors have disappeared after I updated to the newest version, but need to double-check this to be sure..

**Comment by Schepp** [comment]
Is it still there? If yes: it is the the restore function trying to restore DOM elements that might already be deleted at that point (due to reload)

**Comment by olemarius** [comment]
Yep it's still there. And I updated to the current version on GitHub before testing now..

**Comment by Schepp** [comment]
Removed the error for the new version I just pushed. Should keep your console sane, now.
